README.md

```diff
@@ -11,6 +11,37 @@ It is backed by [hmpps-incident-reporting-api](https://github.com/ministryofjust
 
 The application needs a suite of services to work.
 
+### Using services in `dev` environment
+
+This is the easiest way to run and develop on your machine: by hooking into services that already exist
+in the `dev` environment.
+
+Run the application in development mode:
+
+* in separate shell sessions, for local development
+  * This will automatically restart it if server code or front-end assets are modified.
+
+### Using only local services
+
+```shell
+docker compose -f docker-compose.yml up -d
+npm run start:dev-local
```

Comment:
Calling this gives me

```
> hmpps-incident-reporting@0.0.1 start:dev-local
> export $(cat local.env) && concurrently -k -p "[{name}]" -n "ESBuild,Node" -c "yellow.bold,cyan.bold" "node esbuild/esbuild.config.js --build --watch" "node esbuild/esbuild.config.js --dev-server"

sh: concurrently: command not found
```

if there is something to install first, should it be listed in the README as a prerequisite?

Reply:
yes, probably obvious but maybe good to add to the README anyway?